### ISSUE TITLE
Assetpath doesn't get prepended for attributes like src

### ISSUE THREAD

**Issue by web-padawan** [Polymer/polymer-bundler#430]
After upgrading to `polymer-build` 0.8.0 and `polymer-bundler` 2.0.0-pre.5 I have bugs with assetpath attribute.

I created a [repro](https://github.com/web-padawan/polymer-sandbox) using `gold-cc-cvc-input`.  See it in action [here](https://sandbox-e0d4b.firebaseapp.com/) (have a look at the right side of the input, where the image should be).

Please pay attention to that my case is a bit tricky, since I have `vendor` outside of the `src`, one level up than `index.html`. Previously, I used to fix that for my project by stripping out one directory (`../`) from assetpath, like this: 

```
.pipe($.replace(/(assetpath=".+)(\.\.\/vendor)(?=.+")/g, '$1vendor'))
```

@usergenic could you clarify recent changes a bit?

**Comment by web-padawan** [comment]
UPD: created a [repro](https://github.com/web-padawan/polymer-sandbox) using `gold-cc-cvc-input`.  See it in action [here](https://sandbox-e0d4b.firebaseapp.com/) (have a look at the right side of the input, where the image should be).

Please pay attention to that my case is a bit tricky, since I have `vendor` outside of the `src`, one level up than `index.html`. Previously, I used to fix that for my project by stripping out one directory (`../`) from assetpath, like this: 

```
.pipe($.replace(/(assetpath=".+)(\.\.\/vendor)(?=.+")/g, '$1vendor'))
```

**Comment by web-padawan** [comment]
So, below is the completed set of differences between `vulcanize` (before) and `polymer-bundler` (after)

## 1. Shell order

Before: 
```
<head>
    <meta charset="UTF-8">
    <link rel="import" href="../../app/sandbox-app/sandbox-app.html">
</head>

<body>
    <div hidden="" by-vulcanize="">
```

After:

```
<head></head>

<body>
    <div hidden="" by-polymer-bundler="">
        <link rel="import" href="../../app/sandbox-app/sandbox-app.html">
```

## 2. assetpath attribute

Before:
```
<dom-module id="gold-cc-cvc-input" assetpath="../../../../vendor/gold-cc-cvc-input/">
```

After: 
```
<dom-module id="gold-cc-cvc-input" assetpath="../../../vendor/gold-cc-cvc-input/">
```

## 3. src attribute

Before: 
```
                        <iron-icon id="icon" src="../../../../vendor/gold-cc-cvc-input/cvc_hint.png" hidden$="[[_amex]]" alt="cvc"></iron-icon>
                        <iron-icon id="amexIcon" hidden$="[[!_amex]]" src="../../../vendor/gold-cc-cvc-input/cvc_hint_amex.png" alt="amex cvc"></iron-icon>
```

After: 
```
                        <iron-icon id="icon" src="cvc_hint.png" hidden$="[[_amex]]" alt="cvc"></iron-icon>
                        <iron-icon id="amexIcon" hidden$="[[!_amex]]" src="cvc_hint_amex.png" alt="amex cvc"></iron-icon>
```

## 4. entrypoint assetpath 

Before: 
```
 <dom-module id="sandbox-demo-first-sub-page">
```

After: 
```
 <dom-module id="sandbox-demo-first-sub-page" assetpath="./">
```

**Comment by usergenic** [comment]
Investigating right now, @web-padawan -  Thanks for your patience.

**Comment by usergenic** [comment]
The main problem is that the code that rewrites urls in element attributes is not descending into `<template>` tags to do its job.  Fixed bug in bacbac44aed0242d13da0ddc22f715b70f49f431 which is awaiting a larger PR #431 to land shortly as prerequisite.

**Comment by usergenic** [comment]
As an aside, code in 2.0.0-pre.5 that wrote assetpath as "./" is not doing that anymore, since it is redundant.

**Comment by usergenic** [comment]
I don't see what copies vendor from `./vendor` into `./dist/vendor` for serving to work, but once I do that with the patched version of bundler the demo works.  :)

**Comment by usergenic** [comment]
This is going be released in 2.0.0-pre.6

**Comment by web-padawan** [comment]
@usergenic thanks! Today I finally switched our production build pipeline to `polymer-build` 0.8.1, which decreased bundling time from 2 minutes to just 17 seconds :1st_place_medal:
